packages/core/contracts/financial-templates/common/FundingRateApplier.sol

```diff
@@ -87,6 +90,36 @@ abstract contract FundingRateApplier is Testable, Lockable {
         _getFundingRateStore().setRewardRate(address(this), _fundingRateRewardRate);
     }
 
+    /**
+     * @notice Returns the cumulative funding rate multiplier that would be used on any contract interaction
+     * calling `_getFundingRateAppliedTokenDebt()`. This does not actually update the multiplier, but is a helper
+     * method for monitoring bots and sponsors to use to get the current multiplier.
+     * @dev Calling this method is equivalent to calling `_applyEffectiveFundingRate()` followed by
+     * `cumulativeFundingRateMultiplier()`. Note that an emergency shutdown freezes the funding rate multiplier,
+     * meaning that the caller should instead use the value of `cumulativeFundingRateMultiplier`, which is the final
+     * update for the funding rate multiplier before it is shutdown.
+     * @return newCumulativeFundingRateMultiplier The current funding rate multiplier.
+     */
+    function getCumulativeFundingRateMultiplier()
+        public
+        view
+        returns (FixedPoint.Unsigned memory newCumulativeFundingRateMultiplier)
+    {
+        // If contract is emergency shutdown, return latest multiplier from shutdown time.
+        if (emergencyShutdownTimestamp != 0) {
+            newCumulativeFundingRateMultiplier = cumulativeFundingRateMultiplier;
```

Comment:
this must be a solidity thing but nothing is being returned. how does this syntax work?

Reply:
`newCumulativeFundingRateMultiplier` is declared in the function name declaration in the `returns()` block